k8s/wordpress/manifest/manifests.yaml.template

```diff
@@ -114,6 +114,15 @@ spec:
     metadata:
       labels: *WordPressDeploymentLabels
     spec:
+      initContainers:
+        - name: deny-access-before-configuration
+          image: $IMAGE_WORDPRESS
+
+          command: ["sh", "-c", "ipadm=$WPADMINISTRATOR_IP_ADDRESS; wpadm=/var/www/html/wp-admin/; if [[ -d $wpadm ]]; then exit; fi; mkdir -m755 $wpadm; chown www-data:www-data $wpadm; echo 'Require all denied\n<If \"%{HTTP_USER_AGENT} =~ /^(kube-probe|GoogleHC)/ && %{HTTP:X-Forwarded-For} =~ /^$/\">\n  Require all granted\n  DirectoryIndex /wp-admin/install.php\n</If>\n<If \"%{HTTP:X-Forwarded-For} =~ /^'${ipadm:=127.0.0.1}'/\">\n  Require all granted\n</If>' >> $wpadm/.htaccess"]
```

Comment:
Could you move this to [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) and add comments with explanation what is going here?